### PR TITLE
changes for related video articles show if coming from home

### PR DESCRIPTION
**Pull request by Kalyani790**
changes for related video articles show if coming from home

### CODE REVIEW

File: app/screens/home/DetailsScreen.tsx [hunk 1]
```diff
@@ -514,6 +535,10 @@ console.log(videoIsFocused,"..videoIsFocused");
               fromScreen === "Home" ?
               <>
               <TrackMilestoneView currentSelectedChildId={currentSelectedChildId}/>
+              <FlexCol style={{ backgroundColor: newBackgroundColor }}>
+                    <RelatedArticles relatedArticles={detailDataToUse?.related_articles} category={detailDataToUse?.category} fromScreen={fromScreen} currentId={detailDataToUse?.id} headerColor={newHeaderColor} backgroundColor={newBackgroundColor} listCategoryArray={listCategoryArray} navigation={navigation} currentSelectedChildId={currentSelectedChildId} />
+                    <RelatedVideoArticles relatedArticles={detailDataToUse?.related_video_articles ? detailDataToUse?.related_video_articles : []} category={detailDataToUse?.category} fromScreen={fromScreen} currentId={detailDataToUse?.id} headerColor={newHeaderColor} backgroundColor={newBackgroundColor} listCategoryArray={listCategoryArray} navigation={navigation} currentSelectedChildId={currentSelectedChildId} />
+              </FlexCol>
```
Comment (Akhror) [inline app/screens/home/DetailsScreen.tsx:541]: Can we combine like: 
            {
              fromScreen === "Home" || fromScreen === "ChildDevelopment" ?
                <>
                  <TrackMilestoneView currentSelectedChildId={currentSelectedChildId} />
                  <FlexCol style={{ backgroundColor: newBackgroundColor }}>
                    ...
                  </FlexCol>
                </> : null
            }

File: app/screens/home/DetailsScreen.tsx [hunk 2]
```diff
@@ -106,6 +107,26 @@ const DetailsScreen = ({route, navigation}: any): any => {
...
+          },
+          merge: true
+        })
+      }
+      else{
+        // navigation.navigate({
+        //   name: fromScreen == "ChildgrowthTab2" ? "ChildgrowthTab" : fromScreen,
+        //   params: {categoryArray:listCategoryArray,backClicked:'yes'},
+        //   merge: true,
+        // });
```
Comment (Akhror) [inline app/screens/home/DetailsScreen.tsx:126]: Let's please remove the comment if not neccessary.

Comment (Kalyani790) [reply app/screens/home/DetailsScreen.tsx:126]: ok

File: app/screens/home/DetailsScreen.tsx (hunk 1 above)
Comment (Kalyani790) [reply app/screens/home/DetailsScreen.tsx:536]: done

**Pull request merged**